Include/datetime.h

```diff
@@ -154,13 +154,18 @@ typedef struct {
     PyTypeObject *TimeType;
     PyTypeObject *DeltaType;
     PyTypeObject *TZInfoType;
+    PyTypeObject *TimeZoneType;
```

Comment:
@abalkin If you don't want the `_Check` macros exposed, do you also not want this exposed?

Reply:
yes